tests/console/zbar.pm

```diff
@@ -19,7 +19,7 @@ use utils;
 
 sub run {
     select_console 'user-console';
-    assert_script_sudo("zypper -n in zbar");
+    assert_script_sudo("zypper -n in wget zbar");
```

Comment:
Also here, might be more explicit to install "wget" in its own call to make the test dependency more clear

Reply:
Not really - the assert not only means "verify that the product works" but also "verify that the test can continue".

I don't see any value in separating it.